PhotoStream/app/src/main/java/com/keenanzucker/photostream/DatabaseService.java

```diff
@@ -0,0 +1,46 @@
+package com.keenanzucker.photostream;
+
+import android.content.Context;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+
+/**
+ * Created by keenan on 10/2/15.
+ * Database service class sets up the SQL database and creates the simple table that holds
+ * image URLs.
+ * Also deals with upgrading and downgrading of database.
+ */
+
+
+public class DatabaseService extends SQLiteOpenHelper {
+    // If you change the database schema, you must increment the database version.
+    // If you change the database schema, you must increment the database version.
+    public static final int DATABASE_VERSION = 1;
+    public static final String DATABASE_NAME = "FeedReader.db";
+
+    public DatabaseService(Context context){
+        super(context, DATABASE_NAME, null, DATABASE_VERSION);
+    }
+
+    public void onCreate(SQLiteDatabase db){
+        final String TEXT_TYPE = " TEXT";
+        final String SQL_CREATE_ENTRIES =
```

Comment:
It might be nicer to define these within your contract file with the rest of the parameters.